### ISSUE TITLE
Remove Hobby Shop preset

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7882]
...because no one knows what it is. See Wiki: https://wiki.openstreetmap.org/wiki/Tag:shop%3Dhobby
My hobby is OpenStreetMap - would I buy a map in a Hobby Store?:)

This preset has terms: manga, figurine, model - we already have separate presets for these ones - `shop=anime`, `shop=collector` and `shop=model`.

I suggest addding "hobby" as a term for these presets:

* `shop=anime`
* `shop=bicycle`
* `shop=boat`
* `shop=books`
* `shop=brewing_supplies`
* `shop=collector`
* `shop=craft`
* `shop=doityourself`
* `shop=fishing`
* `shop=games`
* `shop=hunting`
* `shop=model`
* `shop=music`
* `shop=musical_instrument`
* `shop=outdoor`
* `shop=scuba_diving`
* `shop=sports`
* `shop=toys`
* `shop=video`
* `shop=video_games`

So one can easily find the right preset.

**Comment by quincylvania** [comment]
@maro-21 I agree people should use more specific tags if they exist. But it's also useful to have a general category for some cases, as evidenced by the hundreds of uses of the tag. Some shops brand themselves as hobby shops after all—there are [five entries in the NSI](https://nsi.guide/index.html?k=shop&v=hobby).

Anyway, you'll need to get more buy-in from the community before we can remove an existing preset like this. You can't just edit the wiki and open an iD issue 😄

**Comment by maro-21** [comment]
> You can't just edit the wiki and open an iD issue

To be clear - the only thing I did was expanding the list of suggested replacements:
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&diff=2017292&oldid=1806009
Everyone can see how the Wiki page looked like before my edit:
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&action=history
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&oldid=1806009

> when the hobby presets/brands were added to iD/NSI last year, there was no discussion [cd1a842](https://github.com/openstreetmap/iD/commit/cd1a842cb81f3496a57eabe8ac3548a427c784b0)

Exactly :)


> Some shops brand themselves as hobby shops after all—there are [five entries in the NSI](https://nsi.guide/index.html?k=shop&v=hobby).

This has been fixed: https://github.com/osmlab/name-suggestion-index/commit/59d701f2e0d87da49dc2abe79d459eebbdc43cce
